app/feature/feature-help-center/src/main/kotlin/com/hedvig/android/feature/help/center/data/GetMemberActionsUseCase.kt

```diff
@@ -0,0 +1,106 @@
+package com.hedvig.android.feature.help.center.data
+
+import arrow.core.Either
+import arrow.core.raise.either
+import com.apollographql.apollo3.ApolloClient
+import com.hedvig.android.apollo.safeExecute
+import com.hedvig.android.apollo.toEither
+import com.hedvig.android.core.common.ErrorMessage
+import com.hedvig.android.featureflags.FeatureManager
+import com.hedvig.android.featureflags.flags.Feature
+import com.hedvig.android.logger.logcat
+import kotlinx.coroutines.flow.first
+import kotlinx.serialization.Serializable
+import octopus.MemberActionsQuery
+
+internal interface GetMemberActionsUseCase {
+  suspend fun invoke(): Either<ErrorMessage, MemberAction>
+}
+
+internal class GetMemberActionsUseCaseImpl(
+  private val apolloClient: ApolloClient,
+  private val featureManager: FeatureManager,
+) : GetMemberActionsUseCase {
+  override suspend fun invoke(): Either<ErrorMessage, MemberAction> {
```

Comment:
For this, can you try to use parZip here from arrow.fx?
Basically we want to find all those 4 things, but it would be nice to kick all of those off in parallel, so that we do not ask for one, wait for it to return, ask the next one, wait for that too, and so on.

I just did this locally on my machine to see how it would compile. And I created a git patch out of this which I have sent to you in Slack, check it out there.
Try to copy that and in your IDE find the "Apply patch..." action to apply it. Did this since it requires you to add a dependency too so I couldn't just give an inline suggestion here.